testing/test_warnings.py

```diff
@@ -8,7 +8,6 @@
 
 import pytest
 
-
```

Comment:
Strange, are those whitespace changes necessary?

Reply:
If isort and `reorder-python-imports` differ in the output, won't it cause problems for isort users, meaning their commits will "fix" the files again? If so this defeats the purpose of having this support in place anyway, which is to avoid commit failures due to import order.

Reply:
> Strange, are those whitespace changes necessary?

No, but they are consistent now (1 line after imports).
It's just that reorder-python-imports does not care.


> If isort and `reorder-python-imports` differ in the output

They don't anymore with this config.
I've done this since I am using isort myself, and was annoyed by it e.g. not putting future imports on single lines etc.

Reply:
If reoder-python-imports would complain CI would have failed.

Reply:
True, when I commented out I didn't look at the CI status. 👍